pfdicom_rev/pfdicom_rev.py

```diff
@@ -583,7 +670,27 @@ def processJSON(self, **kwargs):
                             persistAnalysisResults  = False
         )
         return d_process
-
+    
+    def processMAP(self, **kwargs):
+        """
+        A simple "alias" for calling the pftree method.
+        """
+        d_process       = {}
+        d_process       = self.pf_tree.tree_process(
+                            inputReadCallback       = self.inputReadCallbackMAP,
+                            analysisCallback        = self.inputAnalyzeCallbackMAP,
+                            outputWriteCallback     = self.outputSaveCallbackMAP,
+                            persistAnalysisResults  = False
+        )
+        d_library = {
+            "data": list(self.pf_tree.d_inputTree.keys())
+            }
+        self.dp.qprint("analyzing: %s" % d_library, level = 1)
+        with open("%s/map.json" % self.str_inputDir, 'w') as outfile:
+            json.dump(d_library, outfile)
```

Comment:
Please add the  'indent =4' here so that the output file is indented and easily human readable.